kinto/core/resource/schema.py

```diff
@@ -6,7 +6,9 @@
                                FieldList, TimeStamp, URL)
 from kinto.core.utils import native_value
 
-POSTGRESQL_MAX_INTEGER_VALUE = 9223372036854775807
+POSTGRESQL_MAX_INTEGER_VALUE = 2**64 / 2
+
+positiveBigInteger = colander.Range(min=0, max=POSTGRESQL_MAX_INTEGER_VALUE)
```

Comment:
Hum, now I don't know anymore :D 

A constant seems weird for a reusable instance no ? 
Maybe like a class name ? Like `colander.Length` ? Or just `positive_integer` like a variable?